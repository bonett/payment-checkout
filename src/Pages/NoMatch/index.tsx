import React from "react";
import { Link } from "react-router-dom";

function NoMatch() {

  return (
    <div className="not-found">
      <h2>404: Page Not Found</h2>
      <Link to={'/'}>Go Home</Link>
    </div>
  );
}

export default NoMatch;
