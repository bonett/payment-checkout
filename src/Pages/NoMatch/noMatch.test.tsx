import React from 'react';
import { render, screen } from '@testing-library/react';
import NoMatch from './';

test('renders learn react link', () => {
  render(<NoMatch />);
  const linkElement = screen.getByText(/404: Page Not Found/i);
  expect(linkElement).toBeInTheDocument();
});
