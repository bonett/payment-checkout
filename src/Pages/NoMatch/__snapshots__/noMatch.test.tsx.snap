// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not found page Should be render page 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="not-found"
    >
      <h2
        class="not-found_title"
      >
        Page Not Found
      </h2>
      <a
        href="/"
      >
        Go Home
      </a>
    </div>
  </div>
</div>
`;

exports[`Not found page Should go home button clicked 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="not-found"
    >
      <h2
        class="not-found_title"
      >
        Page Not Found
      </h2>
      <a
        href="/"
      >
        Go Home
      </a>
    </div>
  </div>
</div>
`;
